src/main/java/com/google/sps/servlets/AccountData.java

```diff
@@ -0,0 +1,104 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns all account related info (links to login pages/account info). */
+@WebServlet("/account")
+public class AccountData extends HttpServlet {
+
+  private final class JsonAccountInfo {
+    private final boolean loggedIn;
+    private final String logInOutURL;
+    private final String signUpURL;
+
+    public JsonAccountInfo(Boolean loggedIn, String logInOutURL, String signUpURL) {
+      this.loggedIn = loggedIn;
+      this.logInOutURL = logInOutURL;
+      this.signUpURL = signUpURL;
+    }
+
+    public boolean getLoggedIn() {
+      return loggedIn;
+    }
+
+    public String getLogInOutURL() {
+      return logInOutURL;
+    }
+
+    public String getSignUpURL() {
+      return signUpURL;
+    }
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
```

Comment:
No need to change now, but going forward please try to make the code tighter by only adding empty lines when it helps readability.